generic/ras.py

```diff
@@ -44,12 +45,12 @@ def setUp(self):
         if "ppc" not in architecture:
             self.skip("supported only on Power platform")
         sm = SoftwareManager()
-        for package in ("ppc64-diag", "powerpc-utils", "lsvpd"):
+        for package in ("ppc64-diag", "powerpc-utils", "lsvpd", "ipmitool"):
             if not sm.check_installed(package) and not sm.install(package):
                 self.error("Fail to install %s required for this"
                            " test." % package)
 
-    @skipIf(IS_POWER_NV, "Skipping test in PowerNV platform")
+    @skipIf(IS_POWER_NV or IS_KVM_GUEST, "Skipping test in PowerNV platform")
```

Comment:
Skip message does not specify that KVM Guest is also unsupported. Please change it in all places.